pkg/proxy/docker/docker.go

```diff
@@ -419,22 +419,24 @@ type patchSet struct {
 
 // ContainerTruststorePatcher provides a Docker API proxy that patches the container truststore while running.
 type ContainerTruststorePatcher struct {
-	cert            x509.Certificate
-	envVars         []string
-	javaEnvVar      bool
-	networkOverride string // TODO: Not a good fit for this abstraction
-	proxySocket     string
-	patchMap        map[string]*patchSet
-	m               sync.Mutex
-	created         atomic.Uint32
+	cert              x509.Certificate
+	envVars           []string
+	trustStoreEnvVars []string
+	javaEnvVar        bool
+	networkOverride   string // TODO: Not a good fit for this abstraction
+	proxySocket       string
+	patchMap          map[string]*patchSet
+	m                 sync.Mutex
+	created           atomic.Uint32
 }
 
 // ContainerTruststorePatcherOpts defines the optional parameters for creating a ContainerTruststorePatcher.
 type ContainerTruststorePatcherOpts struct {
-	EnvVars         []string
-	JavaEnvVar      bool
-	RecursiveProxy  bool
-	NetworkOverride string
+	EnvVars           []string
+	TrustStoreEnvVars []string
```

Comment:
nit: I think here we're considering "truststore" as one word so the capitalization should reflect that